lists/eg.csv

```diff
@@ -684,3 +684,143 @@ https://proximize.me,ANON,Anonymization and circumvention tools,2018-03-26,AFTE,
 https://www.myprivateproxy.net,ANON,Anonymization and circumvention tools,2018-03-26,AFTE,
 http://proxy.dizinc.com,ANON,Anonymization and circumvention tools,2018-03-26,AFTE,
 http://www.proxymesh.com,ANON,Anonymization and circumvention tools,2018-03-26,AFTE,
+http://www.akhbarelyom.com/,NEWS,News Media,2018-02-09,OONI,
+https://amp.cloudflare.com/v0/amp-iframe-0.1.js,HOST,Hosting and Blogging Platforms,2018-02-09,OONI,
+https://cdn.ampproject.org/v0/amp-iframe-0.1.js,HOST,Hosting and Blogging Platforms,2018-02-09,OONI,
```

Comment:
Can you drop these two as I am going to merge these into the global list in: https://github.com/citizenlab/test-lists/pull/300